br/pkg/lightning/backend/local/duplicate.go

```diff
@@ -579,6 +584,29 @@ func (m *DuplicateManager) buildDupTasks() ([]dupTask, error) {
 	return tasks, nil
 }
 
+func (m *DuplicateManager) buildIndexDupTasks() ([]dupTask, error) {
```

Comment:
Could you please add a unit test for this function? (I'm so sorry there is no test for this file before)

Reply:
Done.